### ISSUE TITLE
How reversible is our use of Airtable?

### ISSUE THREAD

**Issue by maelle** [ropensci-org/buffy#35]
How easy would it be to swap Airtable for another service?

Cc @mpadge @xuanxu

**Comment by xuanxu** [comment]
The code connecting with Airtable is really tied to their API so it can not be reused for other service. The good thing is that all the Airtable-related code is in [a single place](https://github.com/ropensci-org/buffy/blob/ropensci/app/workers/ropensci/airtable_worker.rb), so to use another service we would need to create just one object, replicating the functionality but using a different API. 

In short: The change is not straightforward but should be easy enough.

**Comment by maelle** [comment]
Thank you, that is good to know and seems doable indeed!
